holoviews/element/graphs.py

```diff
@@ -478,6 +478,66 @@ def from_networkx(cls, G, positions, nodes=None, **kwargs):
         # Construct graph
         return cls((edge_data, nodes), vdims=edge_vdims)
 
+    @classmethod
+    def from_sparse(cls, edges, nodes, **params):
+        """Create a Graph element from a sparse adjacency array.
+
+        Parameters
+        ----------
+        edges : scipy.sparse array
```

Comment:
or matrix I assume?

Reply:
I think there is no universally agreed metaterm except for the clunky “sparse container”